### ISSUE TITLE
Crosswalk repositories are missing

### ISSUE THREAD

**Issue by dgarijo** [codemeta/codemeta#228]
We did a mapping between OntoSoft and codemeta, and it's available on: https://github.com/codemeta/codemeta/blob/master/crosswalks/OntoSoft.csv

However, this does not appear anymore on the main crosswalk page. Is there a reason for this? I have also mapped https://w3id.org/okn/o/sd to codemeta and provided the crosswalk here: https://docs.google.com/spreadsheets/d/1QvMiDZhWl_xYI3p-vFZuQpYkbUoxbQlrRiNAFQwlNnA/edit#gid=470365598 It would be great to show this in the page too. I can do a pull request if needed.

**Comment by dgarijo** [comment]
I noticed that other repos are missing too, such as ASCL

**Comment by mbjones** [comment]
What do you mean by the main crosswalk page?  Are you referring to: https://codemeta.github.io/crosswalk/ ?  If so, then I think its been that way all along, and @cboettig just picked out some to highlight there.  I've always thought it odd that we didn't list all of the crosswalks there that show up in the full crosswalks table that is linked on that page.  It would be nice to add them all in.  Maybe @cboettig has thoughts on why its how it is now...

**Comment by dgarijo** [comment]
@mbjones, yes, I am referring to https://codemeta.github.io/crosswalk/. Since we have done the effort to map the vocabularies, it would be great to show it up on the page. In addition, it shows that many more people/organizations are supporting Codemeta :)

**Comment by cboettig** [comment]
Thanks @dgarijo !  yeah, the webpage generation isn't entirely automated.  

- There's the (now separate) crosswalks for each vocabulary in the `/crosswalks` dir as you linked.
- There are some python scripts to automatically combine these into the combined crosswalk listed in the repo root, but that script is not run automatically (e.g. we don't have it on a CI/CD setup for when new crosswalks are added or updated) 

Then on top of that, the website side is also somewhat manual.  Website is built in hugo / blogdown, and has some manually written .Rmd files in `content/` which generate the pages shown on  the website (using the data from the combined crosswalk table, currently).  Ideally those source pages would be generated automatically, and then the hugo site rebuilt and rendered automatically, but for the moment that's not been implemented.  

We know the website could use some other love too, (e.g. https://github.com/codemeta/codemeta.github.io/issues/20, etc).  PRs for the website are more than welcome!

**Comment by cboettig** [comment]
(and just to be clear, I didn't intend to signal any special status to the ones already on the website -- as you can probably see from the git history, those were just the ones that were already in the crosswalk when I was putting the site together ~ 3 years ago!  It's been great to get so many more community contributions since then!  CodeMeta really is a community owned project)

**Comment by dgarijo** [comment]
All right. In the workshop we have pushed a bunch of new crosswalks. I think we should make all of them available. Probably Tom or Alice will do a PR soon. Otherwise at least I will push mine :)
